scripts/verify_tla_bridge_coverage.py

```
@@ -0,0 +1,204 @@
+            inv, prop = parse_cfg(cfg)
+            all_inv.extend(inv)
+            all_prop.extend(prop)
+        all_inv = list(dict.fromkeys(all_inv))
+        all_prop = list(dict.fromkeys(all_prop))
+
+        if not actions and not all_inv and not all_prop:
+            continue
+
+        print(f"{basename}:")
+        t, b = _print_section("Actions", actions, hits)
+        tot_a += t; br_a += b
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)